enterprise/lang-js/src/main/java/io/crate/operation/language/PolyglotValuesConverter.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * This file is part of a module with proprietary Enterprise Features.
+ *
+ * Licensed to Crate.io Inc. ("Crate.io") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ *
+ * Unauthorized copying of this file, via any medium is strictly prohibited.
+ *
+ * To use this file, Crate.io must have given you permission to enable and
+ * use such Enterprise Features and you must have a valid Enterprise or
+ * Subscription Agreement with Crate.io.  If you enable or use the Enterprise
+ * Features, you represent and warrant that you have a valid Enterprise or
+ * Subscription Agreement with Crate.io.  Your use of the Enterprise Features
+ * if governed by the terms and conditions of your Enterprise or Subscription
+ * Agreement with Crate.io.
+ */
+
+package io.crate.operation.language;
+
+import io.crate.data.Input;
+import io.crate.types.ArrayType;
+import io.crate.types.DataType;
+import io.crate.types.DataTypes;
+import io.crate.types.GeoPointType;
+import io.crate.types.GeoShapeType;
+import io.crate.types.ObjectType;
+import org.graalvm.polyglot.TypeLiteral;
+import org.graalvm.polyglot.Value;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+class PolyglotValuesConverter {
```

Comment:
This and `resolvePolyglotFunctionValue` can be moved to the udf package of the sql module once we add support for more languages.